crypto/evp/ec_support.c

```diff
@@ -150,6 +150,7 @@ int ossl_ec_curve_name2nid(const char *name)
 /* Functions to translate between common NIST curve names and NIDs */
 
 static const EC_NAME2NID nist_curves[] = {
+#ifndef OPENSSL_NO_EC2M
```

Comment:
These names are used outside of FIPS module for mappings. And in theory there might be third party providers that support binary curves in NO_EC2M builds. However yes, if built inside the FIPS provider you can add this condition.

```suggestion
#if !defined(OPENSSL_NO_EC2M) || !defined(FIPS_MODULE)
```

Reply:
Ah! Thanks.

Reply:
So is there an update coming to do as @t8m suggested?